app/libs/asset.py

```diff
@@ -36,22 +36,38 @@ def get_pending_comments(id, parent=0, offset=None, limit=None):
     where = [PendingComment.asset == id, PendingComment.parent == parent]
     if offset is not None:
         where.append(PendingComment.id < offset)
-    comments = PendingComment.select().where(*where).order_by(PendingComment.id.desc())
+    comments = PendingComment.select(
+            PendingComment, Commenter
```

Comment:
@gauravr  I would suggest including only necessary fields for Commenter. For example (bio, web) fields are not needed and for large number of comments commenters, it will increase the response size.

Reply:
@shon Done